operations/network/management_network/snmp_exporter_configs.md

```diff
@@ -0,0 +1,81 @@
+## Prometheus SNMP Exporter 
+The Prometheus SNMP Exporter is deployed by the the `cray-sysmgmt-health` chart to the `sysmgmt-health` namespace as part of the Cray System Management \(CSM\) release.
+
+### Configuration
+
+In order to provide data to the Grafana SNMP dashboards the SNMP Exporter must be configured with a list of management network switches to scrape metrics from.
+
+1. Set the `SYSTEM_NAME` variable if not already set.
+
+```bash
+linux# SYSTEM_NAME=eniac
+```
+
+1. Obtain the list of switches to use as targets using CANU (CSM Automatic Network Utility)
```

Comment:
- All of the steps below are missing a period at the end of the line. Please add one to each step.
- The spelled out acronym name should be listed first: CSM Automatic Network Utility (CANU)

Reply:
done